plugin/trino-smartnews-functions/src/main/java/io/trino/plugin/smartnews/functions/actions/ChooseNotInterestedParseFunctions.java

```diff
@@ -67,7 +67,7 @@ public ChooseNotInterestedParseFunctions()
 
     @Description("parse action chooseNotInterested")
     @ScalarFunction("parse_choose_not_interested")
-    @SqlType("row(device_token varchar, virtual_id varchar, edition varchar, platform varchar, session_id varchar, app_version varchar, system_version varchar, connection_type varchar, locale varchar, timezone varchar, hardware varchar, event_timestamp bigint, account_id varchar, user_creation_timestamp bigint, ab_test array(map(varchar, integer)), ab_test_v3_variant_ids array(integer), " +
+    @SqlType("row(device_token varchar, virtual_id varchar, edition varchar, platform varchar, session_id varchar, app_version varchar, system_version varchar, connection_type varchar, locale varchar, timezone varchar, hardware varchar, event_timestamp bigint, account_id varchar, user_creation_timestamp bigint, ab_test array(map(varchar, integer)), ab_test_v3_variant_ids array(integer), app_id varchar, " +
```

Comment:
the code change looks good to me, thanks.
However, 3 comments:
1, this repo is deprecated, please make you change based on branch `smartnews_435` in repo `trinodb`, https://github.com/smartnews/trinodb/tree/smartnews_435, your access is just granted.
2, update the version number of dmp-core-java in trinodb/plugin/trino-smartnews-functions/pom.xml, to make sure ActionBase does have the app_id field.
3, please also add tests to cover the change, currently the tests are located at TestNewsActionsParseFunctions.java

Reply:
also, please remember to run mvn package, to capture any code style error, though the first time, it will take you some time.